apps/src/lib/cli.rs

```diff
@@ -987,6 +1004,50 @@ pub mod cmds {
         }
     }
 
+    #[derive(Clone, Debug)]
+    pub struct TxNftCreate(pub args::NftCreate);
+
+    impl SubCmd for TxNftCreate {
+        const CMD: &'static str = "nft-create";
```

Comment:
tiny nit, but just to be consistent with `init-account` and `init-validator`, I'd name this one:
```suggestion
        const CMD: &'static str = "init-nft";
```